react/react1/week3/todo-week3/src/todoweek3/ListTodo.js

```diff
@@ -0,0 +1,95 @@
+import { useState, useEffect } from "react";
+import ItemTodo from "./ItemTodo";
+import { FancyBorder } from "./FancyBorder";
+const urlApi =
+  "https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw";
+export function ListTodo() {
+  const [description, setDescription] = useState("");
```

Comment:
Try to somehow separate the part where you add a new tod from the to list.
Adding a todo is not a todo list

Reply:
Hi, Thanks. I modified it to another component. 👍